packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -72,7 +72,7 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
     shallow,
   );
 
-  const { canChangePasswordWithOldPassword, canExportAccountData, canDeleteAccount } = accountController.getFeatures();
+  const { canChangePasswordWithOldPassword, canExportAccountData } = accountController.getFeatures();
```

Comment:
Do we still need "canExportAccountData"?

Reply:
01a95c9ceda9cf3127a3fe29a2af2a8109642d2e